inspirehep/modules/migrator/models.py

```diff
@@ -31,7 +31,8 @@
 from sqlalchemy.ext.hybrid import hybrid_property
 
 
-class InspireProdRecords(db.Model):
+class LegacyRecordsMirror(db.Model):
+    # the model has a different name from the table for compatibility
     __tablename__ = 'inspire_prod_records'
```

Comment:
It's unfortunate to get the table names unsynced after, what would be needed to change it too?

Reply:
an alembic script migrating the tables, which adds a bit of complexity.